src/styles/utils/_loops.scss

```
@@ -0,0 +1,27 @@
+@for $index from 1 through 31 {
+  .calendar__block:nth-child(#{$index})::before {
+    content: "#{$index}";
+  }
+}
+
+@for $index from 28 through 31 {
+  .calendar--month-length--#{$index}
+  .calendar__block:nth-child(n + #{$index + 1}) {
+    opacity: 0%;
+  }
+}
+
+$days-of-week:
+  "sun" 0,
+  "mon" $square-size + $gap,
+  "tue" $square-size * $tue-position + $gap * $tue-position,
```

save only position here, do the calculation in the loop, don't repeat yourself
```suggestion
  "tue" $tue-position,
```